pyntc/devices/ios_device.py

```diff
@@ -582,7 +681,7 @@ def set_boot_options(self, image_name, **vendor_specifics):
         try:
             command = "boot system {0}/{1}".format(file_system, image_name)
             self.config_list(["no boot system", command])
-        except CommandError:
+        except CommandListError:
```

Comment:
Why two different errors, one for command and one for commandList?

Reply:
CommandListError gives additional information explaining what command were sent before a failure was incurred.

Add for this update, this is actually the correct implementation for the existing code, as `config_list` raises this exception. It might still work, since `CommandListError` is raised when `CommandError` is raised.

Reply:
Got it. Thanks for the response @jmcgill298